Error 1 (14,6): 'x' ya existe en este ambito
